Description: Sets the Java language level to build with Java 11
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/build.gradle
+++ b/build.gradle
@@ -60,8 +60,8 @@
     apply plugin: 'java'
 
     buildDir = 'target'
-    sourceCompatibility = 1.6
-    targetCompatibility = 1.6
+    sourceCompatibility = 7
+    targetCompatibility = 7
 
     group = 'org.codehaus.groovy'
     version = groovyVersion
